### ISSUE TITLE
API

### ISSUE THREAD

**Issue by lebronjamesnum5** [evansloan/collectionlog.net#69]
When trying to view collection logs on the website through the runelite plugin, I receive "Error connecting to api.collectionlog.net" and am unable to view anything. Is there a fix to this?

**Comment by Napokue** [comment]
Hi @lebronjamesnum5, does this problems still persists for you? If so, can you please provide me usernames you were looking up that caused this error to display so we can investigate. Thanks!

**Comment by lebronjamesnum5** [comment]
It's all of them. I cant view any logs.

**Comment by lebronjamesnum5** [comment]
Incognito fixes it. Is there a way to know which plugin is causing the probelm?

**Comment by Napokue** [comment]
Yes, there are two approaches I personally would take to find this plugin:
- Disabling each plugin one by one and see which one is it. This works if you do not have a lot of plugins installed.
- If you have a lot of plugins installed, first disable the first half and check if the problem is solved. If it is, it is in the first half of the plugins, if not disable the second half. In the half that contains the malfunctioning plugin, do the exact same by one enabling the half of that half and continue to do so until you narrow it down.

**Comment by lebronjamesnum5** [comment]
I only had 3 extensions on chrome and removed all of them and it still didn't work.
